### ISSUE TITLE
Hyperparameters for TabDDPM

### ISSUE THREAD

**Issue by ZinebSN** [amazon-science/tabsyn#15]
Hello @hengruizhang98,
I am getting large errors for column wise density estimation when evaluating TabDDPM on some datasets (Beijing and Magic), did you use different hyperparameters for this model for different datasets? It would be very helpful if you can share the set of hyperparameters used for each model-dataset combination. 
Thanks a lot!

**Comment by ZinebSN** [comment]
For Great, the synthetic data in "occupation" column has a value of 'Local-gov', but this category doesn't exist in the training set, did you just ignore this category in this feature when applying one hot encoding for quality evaluation (alpha precision and beta recall)?

**Comment by hengruizhang98** [comment]
> Hello @hengruizhang98, I am getting large errors for column wise density estimation when evaluating TabDDPM on some datasets (Beijing and Magic), did you use different hyperparameters for this model for different datasets? It would be very helpful if you can share the set of hyperparameters used for each model-dataset combination. Thanks a lot!

We simply use the default hyperparameters of TabDDPM for all datasets.

**Comment by hengruizhang98** [comment]
> For Great, the synthetic data in "occupation" column has a value of 'Local-gov', but this category doesn't exist in the training set, did you just ignore this category in this feature when applying one hot encoding for quality evaluation (alpha precision and beta recall)?

When split the data, please make sure that all categories exist in the training set. If not you can split the data with another seed.

**Comment by ZinebSN** [comment]
@hengruizhang98 Thanks for your replies. I am having issues reproducing some TabDDPM results (for beijing for example) using the default hyperparameters. Would it be possible to rerun them from your end and share the results you get?

**Comment by dylanponsford** [comment]
@ZinebSN It was a long time ago now, but did you ever figure out how to reproduce these results with TabDDPM? I am also having trouble doing so.

Specifically, it seems like the training loss (for the numerical columns) spikes after a certain number of epochs, and then after that, both the numerical loss and categorical loss stay roughly constant and the model stops learning. I think this is behaviour also observed in a later paper ([TabDiff](https://arxiv.org/pdf/2410.20626), Appendix F.2), but there it is only for the larger, more difficult datasets (like news and diabetes), whereas I cannot reproduce the results for 'easier' datasets like adult, beijing, and default.

I saw also that you worked on this later paper @hengruizhang98 , would you be able to provide any insight about this?
